cron/src/index.js

```diff
@@ -0,0 +1,25 @@
+const {
+  getLatestReleaseInFeed,
+  releaseExistsInRepo,
+  notifySlack,
+  triggerTravisBuild,
+} = require('./lib.js');
+
+
+module.exports.wordpressRelease = async (event, context) => {
+  const tag = await getLatestReleaseInFeed();
+  console.log(`Latest wordpress/wordpress release: ${tag}`);
+  const existsInRepo = await releaseExistsInRepo(tag);
+
+  if (!existsInRepo) {
+    console.log(`Release for ${tag} does not in roots/wordpress`);
```

Comment:
"...does not in roots/wordpress" should be "...does not **exist** in roots/wordpress"

Reply:
prs accented